app/src/main/res/layout/topic_play_story_summary.xml

```diff
@@ -19,84 +19,125 @@
+
+            <org.oppia.app.customview.SegmentedCircularProgressView
+              android:id="@+id/story_progress_view"
+              android:layout_width="48dp"
+              android:layout_height="48dp" />
+
+            <TextView
+              android:id="@+id/story_progress_percentage"
+              android:layout_width="wrap_content"
+              android:layout_height="wrap_content"
+              android:layout_gravity="center"
+              android:fontFamily="sans-serif-medium"
+              android:text="@{@string/topic_story_progress_percentage(storyPercentage)}"
+              android:textColor="@color/oppiaPrimaryText"
+              android:textSize="14sp"
+              android:visibility="@{storyPercentage != 0 ? View.VISIBLE : View.GONE}" />
+          </FrameLayout>
+
+          <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:layout_marginTop="4dp"
+            android:fontFamily="sans-serif"
+            android:text="@{storyPercentage != 100? @string/status_in_progress : @string/status_completed}"
+            android:textColor="@color/oppiaPrimaryText"
+            android:textSize="10sp"
+            android:visibility="@{storyPercentage != 0 ? View.VISIBLE : View.GONE}" />
+        </LinearLayout>
+
+        <LinearLayout
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="20dp"
+          android:gravity="center_vertical"
+          android:orientation="vertical">
+
+          <TextView
+            android:id="@+id/story_name_text_view"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:fontFamily="sans-serif-medium"
+            android:onClick="@{viewModel::clickOnStorySummaryTitle}"
+            android:text="@{viewModel.storySummary.storyName}"
+            android:textColor="@color/oppiaPrimaryText"
+            android:textSize="20sp" />
+
+          <TextView
+            android:id="@+id/chapter_count"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="4dp"
+            android:drawableEnd="@{isListExpanded? @drawable/ic_arrow_drop_up_black_24dp : @drawable/ic_arrow_drop_down_black_24dp}"
+            android:fontFamily="sans-serif"
+            android:text="@{@plurals/chapter_count(viewModel.storySummary.chapterCount, viewModel.storySummary.chapterCount)}"
+            android:textColor="@color/oppiaPrimaryText"
+            android:textSize="16sp" />
+        </LinearLayout>
+      </LinearLayout>
+
+      <LinearLayout
+        android:id="@+id/chapter_list_container"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/whiteLight"
```

Comment:
Add padding top and bottom that matches upper portion design. 
![Screenshot_20191113-221346](https://user-images.githubusercontent.com/54615666/68784703-f8442a80-0662-11ea-9c13-464c67410b58.png)
Or please fix this spacing issue.

Reply:
1.Both inside the card bottom and outside card bottom spacing.
2.I do feel the spacing between the items(cards)seems comparatively less than shown in the mock.